libs/partners/qdrant/tests/integration_tests/async_api/fixtures.py

```diff
@@ -0,0 +1,13 @@
+import logging
+from typing import List
+
+from tests.integration_tests.common import qdrant_is_not_running
+
+logger = logging.getLogger(__name__)
+
+
+def qdrant_locations() -> List[str]:
+    if qdrant_is_not_running():
+        logger.warning("Running Qdrant async tests in memory mode only.")
+        return [":memory:"]
+    return ["http://localhost:6333", ":memory:"]
```

Comment:
@efriis, the tests use both, an in-memory and server instance(if running, skipped otherwise).

To run the server, if needed
```
docker run -p 6333:6333 qdrant/qdrant
```

Reply:
In the current state, running `make integration_tests` will work without any additional steps using an in-memory instance.